docs/src/dev-manual/archivematica.md

```diff
@@ -108,5 +115,16 @@ Preservation system value needed for workflow to be Archvimatica specific:
     [preservation]
     taskqueue = "am"
 
+An amssLocationId is needed for AIP download:
+
+    [am]
+    amssLocationId = "e0ed8b2a-8ae2-4546-b5d8-f0090919df04"
+
+!!! note
+
+    The amssLocationId UUID must match the AMSS location ID in the
+    *enduro_storage* database — it does *not* need to match the actual location
+    UUID in the AM Storage Service.
+
```

Comment:
Not sure if we would want to remove this note after using the secret to configure the `amss_location_id`.